src/components/AudioPlayer.vue

```diff
@@ -1,87 +1,122 @@
 <template>
   <div>
-    <v-bottom-sheet
-      transition="slide-y-transition"
+    <v-footer
       v-if="playerActive"
       v-model="playerActive"
-      class="audio-player"
       dark
-      height="100"
-      hide-overlay
-      persistent
+      height="playerHeight"
+      fixed
+      padless
+      style="z-index: 10000;"
     >
       <v-card
+        class="audio-player-card"
         tile
         width="100%"
         :height="playerHeight"
       >
-        <v-progress-linear
-          v-if="!loading"
-          v-model="progress"
-          color="white"
-          class="`my-0 progress-bar"
-          height="3"
-        />
+        <div class="small-viewer">
+          <v-progress-linear
+            v-if="!loading"
+            v-model="progress"
+            color="white"
+            class="progress-bar"
+            height="3"
+          />
 
-        <v-list>
-          <v-list-item dark>
-            <v-list-item-content>
-              <v-list-item-title v-html="sourceFile.title" />
-              <v-list-item-subtitle>
-                <span v-html="sourceFile.author" />
-                <span class="ml-4">{{ timer }} / {{ duration }}</span>
-              </v-list-item-subtitle>
-            </v-list-item-content>
-
-            <v-spacer />
-
-            <v-list-item-icon :class="{ 'mx-5': $vuetify.breakpoint.mdAndUp }">
-              <v-icon
-                v-if="$data.error"
-                :title="$data.error"
-              >
-                mdi-alert
-              </v-icon>
-              <v-btn
-                icon
-                v-else-if="loading"
-                loading
-              />
-              <v-btn
-                v-else-if="playing"
-                icon
-                @click="pause"
-              >
-                <v-icon>
-                  mdi-pause
-                </v-icon>
-              </v-btn>
-              <v-btn
-                v-else
-                icon
-                @click="play"
+          <v-scroll-y-reverse-transition>
+            <v-list>
+              <v-list-item
+                dark
               >
-                <v-icon>
-                  mdi-play
-                </v-icon>
-              </v-btn>
-            </v-list-item-icon>
+                <v-list-item-avatar
+                  tile
+                  size="60"
+                >
+                  <v-img
+                    height="60"
+                    aspect-ratio="1"
+                    bac
+                    gradient="to bottom, rgba(255,255,255,.1), rgba(255,255,255,.5)"
+                    :src="sourceFile.src"
+                  >
+                    <v-icon
+                      size="32"
+                      color="white"
+                      style="opacity: 0.3;
+                        position: absolute;
+                        top: 50%;
+                        left: 50%;
+                        transform: translate(-50%, -50%);"
```

Comment:
Just a suggestion, but perhaps the code will look cleaner and will be easier to maintain by moving more static css to the `<style>` section?